front/types/run.ts

```diff
@@ -36,9 +36,9 @@ export type BlockStatus = {
 };
 
 export type TraceType = {
-  value?: any;
-  error?: string;
-  meta?: any;
+  value: unknown | null;
```

Comment:
we are receiving `null` not undefined from the network I verified + move to unknown, all in all better typing